test/drivers/event_class/event_class.cpp

```diff
@@ -102,29 +102,52 @@ event_test::~event_test()
 /* This constructor must be used with generic tracepoints
  * that must attach a dedicated BPF program into the kernel.
  */
-event_test::event_test(ppm_event_type event_type):
+event_test::event_test(tp_values tp_code):
 	m_tp_set(TP_VAL_MAX, 0)
 {
 	m_current_param = 0;
-	m_event_type = event_type;
 
-	switch(m_event_type)
+	switch(tp_code)
 	{
-	case PPME_PROCEXIT_1_E:
+	case SCHED_PROC_EXIT:
 		m_tp_set[SCHED_PROC_EXIT] = 1;
```

Comment:
```suggestion
		m_tp_set[tp_code] = 1;
```
and remove all first line in each switch case?

Reply:
ty